SafeMode/vi.lproj/Main.strings

```diff
@@ -0,0 +1,6 @@
+
+/* Class = "UITextView"; text = "SpringBoard có vấn đề và bây giờ nó đang ở trong chế độ an toàn.\n\nSileo/libhooker đã không gây ra vấn đề này, mà đã bảo vệ bạn khỏi nó.\n\nNếu như điều này không xảy ra thường xuyên, bạn có thể bấm nút bên dưới để trở về chế độ bình thường một cách an toàn.\n\nNếu như làm vậy vẫn không có tác dụng, hãy gỡ cài đặt một trong những tinh chỉnh của bạn qua Sileo."; ObjectID = "wd3-1g-ED6"; */
+"wd3-1g-ED6.text" = "SpringBoard có vấn đề và bây giờ nó đang ở trong chế độ an toàn.\n\nSileo/libhooker đã không gây ra vấn đề này, mà đã bảo vệ bạn khỏi nó.\n\nNếu như điều này không xảy ra thường xuyên, bạn có thể bấm nút bên dưới để trở về chế độ bình thường một cách an toàn.\n\nNếu như làm vậy vẫn không có tác dụng, hãy gỡ cài đặt một trong những tinh chỉnh của bạn qua Sileo.";
```

Comment:
> SpringBoard có vấn đề và bây giờ nó đang ở trong chế độ an toàn.\n\nSileo/libhooker đã không gây ra vấn đề này, mà đã bảo vệ bạn khỏi nó.\n\nNếu như điều này không xảy ra thường xuyên, bạn có thể bấm nút bên dưới để trở về chế độ bình thường một cách an toàn.\n\nNếu như làm vậy vẫn không có tác dụng, hãy gỡ cài đặt một trong những tinh chỉnh của bạn qua Sileo.

There really isn't a generic term for the second person in Vietnamese so using "bạn" is fine-ish(?), but I'll feel kinda ick if a computer program calls me friend. Here's my suggestion:
> SpringBoard đã gặp phải vấn đề và bây giờ đang ở chế độ an toàn. Sileo/libhooker không gây ra chuyện này, mà nó đã bảo vệ khỏi vấn đề. Nếu như việc này không xảy ra thường xuyên, có thể nhấn nút bên dưới để quay lại chế độ bình thường. Nếu như làm vậy vẫn không có tác dụng, hãy gỡ cài đặt một vài tinh chỉnh trong Sileo.